inbound.go

```diff
@@ -415,6 +420,13 @@ func (response *InboundCallResponse) doneSending() {
 	if span := response.span; span != nil {
 		if response.applicationError || response.systemError {
 			ext.Error.Set(span, true)
+			errorType := appErrorType
+			if response.systemError {
+				errorType = systemErrorType
+				// if the error is a system error, set the error code in span log
+				span.LogKV("error-code", GetSystemErrorCode(response.err))
```

Comment:
Do these system errors map to the errors described here? https://github.com/uber/tchannel/blob/master/docs/protocol.md#error-0xff

Given that these are well defined, short error codes, a span tag with the key (`rpc.tchannel.system_error_code`) would be more appropriate because it is much easier to search span tags compared to logs.

Reply:
+1

Reply:
Updated in the subsequent commit